src/Apps/ArtQuiz/artQuizRoutes.tsx

```diff
@@ -21,12 +22,11 @@ const ArtQuizResults = loadable(() => import("./Routes/ArtQuizResults"), {
 export const artQuizRoutes: AppRouteConfig[] = [
   {
     path: "/art-quiz",
+    hideFooter: true,
```

Comment:
`non-blocking`: I don't believe this does anything. Ideally it would apply to all child routes but I'm pretty sure it's just ignored.

Reply:
It definitely does do something.

Reply:
What does it do then if we have it also on the child routes?